lodash/index.d.ts

```diff
@@ -12335,7 +12335,7 @@ declare module _ {
          * @param value The value to check.
          * @return Returns true if value is null, else false.
          */
-        isNull(value?: any): boolean;
+        isNull(value?: any): value is null;
```

Comment:
Why is `value` optional? There's no reason to call `isNull()` because that's always `false`. Same for `isNil`.